temba/channels/models.py

```diff
@@ -1905,16 +1905,27 @@ def send_smscentral_message(cls, channel, msg, text):
     def send_vumi_message(cls, channel, msg, text):
         from temba.msgs.models import WIRED, Msg
         from temba.contacts.models import Contact
+        from temba.ussd.models import USSDSession
 
         is_ussd = channel.channel_type in Channel.USSD_CHANNELS
         channel.config['transport_name'] = 'ussd_transport' if is_ussd else 'mtech_ng_smpp_transport'
```

Comment:
oh okay, kinda figured when I used my local deployment of vumi, but wasn't sure about the sms part in prod. thx